code/src/index.css

```diff
@@ -1,13 +1,123 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+	margin: 0;
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+	background: #000;
+	color: #fff;
+}
+* {
+	box-sizing: border-box;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+	font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New", monospace;
+}
+
+.wrapper {
+	display: flex;
+	flex-wrap: wrap;
+}
+.wrapper a {
+	width: 25%;
+	text-decoration: none;
+	color: #fff;
+}
+.wrapper a img {
+	width: 100%;
+	height: 100%;
+}
+.container {
+	position: relative;
+}
+
+.movie-details {
+	position: absolute;
+	opacity: 0;
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-end;
+	align-items: flex-start;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	background: rgba(1, 1, 1, 0.64);
+	color: white;
+}
+h1 {
+	margin: 0 0 5% 5%;
+	font-size: 32px;
+	font-weight: bolder;
+}
+p {
+	margin: 0 0 5% 5%;
+	font-size: 16px;
+}
+.container:hover .movie-details {
+	opacity: 1;
+}
+
+.details-container {
+	background-size: cover;
+	min-height: 100vh;
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-end;
+}
+
+.details {
+	display: flex;
+	align-items: flex-end;
+	padding: 50px;
+}
+.movie-details-cover-image {
+	border: 5px solid #fff;
+}
+
+.movie-introduction {
+	margin-left: 20px;
+}
+.details p {
+	max-width: 400px;
+}
+.votes {
+	font-size: 17px;
+	color: red;
+	font-weight: 400;
+	margin-left: 10px;
+}
+button {
+	position: absolute;
+	display: inline-flex;
+	color: #fff;
+	align-items: center;
+	font-weight: 700;
+	font-size: 20px;
+	text-decoration: none;
+	border: 0cm;
+	left: 50px;
+	top: 50px;
+	background: transparent;
+}
+
+span {
+	padding-top: 5px;
+}
+button:hover span {
+	margin-right: 5px;
+}
+
+.not-found {
+	height: 100%;
+	min-height: 100vh;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	background: #000;
+}
+
+.error {
+	font-size: 3em;
 }
```

Comment:
We noticed you don't have any media queries, so it doesn't look so good with mobiles.